pypeman/msgstore.py

```diff
@@ -246,6 +246,8 @@ def __init__(self, path, store_id):
             raise IndexError
 
         with open(os.path.join(self.base_path, id), "rb") as f:
+            # TODO: we might implement an async version, that reads chunkwise for huge files
+            # and adds a sleep(0) after each chunk to allow task switching?
```

Comment:
Or may be we can use a threadpool or use aiofile lib ?

Reply:
True.

I think this is the approach of aiofiles. https://github.com/Tinche/aiofiles.
Explicit sleeps every chunk however would use less resources, be faster and already cover most of the cases.

I wouldn't change anything until this turns out to be a huge issue

Reply:
Fine. Can you just add aiofiles link to comment ?

Reply:
done

Reply:
can close?